tests/serverless/internal/resources/runtimes/python.go

```diff
@@ -170,7 +171,7 @@ def main(event, context):
 func PythonCloudEvent(runtime serverlessv1alpha2.Runtime) serverlessv1alpha2.FunctionSpec {
 	dpd := ``
 
-	src := `import json
+	src := fmt.Sprintf(`import json
```

Comment:
IMO we could have problems with this substitution.
Python also use "%s" as string substitution and it's possible to have conflict between Go and Python substitution.

In this place we change only runtime which is parameter on the end of long python code. It's hard to read.